README.md

```diff
@@ -207,6 +207,7 @@ require'treesitter-context'.setup{
   -- When separator is set, the context will only show up when there are at least 2 lines above cursorline.
   separator = nil,
   zindex = 20, -- The Z-index of the context window
+  throttle_ms = 100, -- Throttle context updates in milliseconds
```

Comment:
Ideally, we should avoid exposing this. I want to keep the config a little as possible.

Reply:
removed!